From 18b3a50240ff1572d7c5b0eef32e9a986eb35f41 Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Wed, 1 Jan 2025 03:21:31 +0100
Subject: [PATCH 04/11] add missing musl_missing.h

	../src/shared/wall.c:53:20: error: '_PATH_UTMPX' undeclared (first use in this function); did you mean 'PATH_MAX'?

Signed-off-by: fossdd <fossdd@pwned.life>
---
 src/shared/wall.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/shared/wall.c b/src/shared/wall.c
index d5900ef01..369b457ec 100644
--- a/src/shared/wall.c
+++ b/src/shared/wall.c
@@ -16,6 +16,7 @@
 #include "user-util.h"
 #include "utmp-wtmp.h"
 #include "wall.h"
+#include "musl_missing.h"
 
 #if ENABLE_UTMP || ENABLE_LOGIND
 
-- 
2.50.1

